package.json

```diff
@@ -59,14 +62,34 @@
     "eslint-plugin-react": "^7.18.3",
     "eslint-plugin-react-hooks": "^2.4.0",
     "gh-pages": "^3.1.0",
-    "prettier": "^1.19.1",
+    "lint-staged": "^10.0.8",
+    "prettier": "^2.0.5",
     "react": "^16.6.0",
     "react-dom": "^16.6.0",
-    "typescript": "^3.8.3"
+    "stylelint": "^13.3.2",
+    "typescript": "^3.8.3",
+    "husky": "^7.0.0"
   },
   "dependencies": {
     "@cmsgov/design-system": "2.7.1",
     "core-js": "^3.6.4",
     "lodash.uniqueid": "^4.0.1"
-  }
+  },
+  "lint-staged": {
+    "*.{js,jsx}": [
```

Comment:
[fix] can you add `ts, tsx` files to this? Currently, I don't think they are being linted

Reply:
Oh wait... My fix request is still valid, but I also noticed that the .prettierrc file in this repo is different than core.
I was testing by changing single quotes to double quotes and they weren't being corrected here, but that's because they aren't a rule.

Is there a plan to standardize rulesets (either in this PR or the future)